cgmanifest.json

```diff
@@ -988,7 +988,7 @@
                 "Type": "other",
                 "Other": {
                     "Name": "Eclipse Temurin",
-                    "Version": "1.8.0_332",
+                    "Version": "17.0.3_7",
```

Comment:
No need to modify java 1.8 version here.

For JDK 1.8, we recommend to use Eclipse Temurin 1.8. For higher version, we use Microsoft Build of OpenJDK.

Reply:
Thanks, I just reverted this change